docs/spec.md

```diff
@@ -2456,14 +2456,16 @@ Performs element-wise exponential operation on `operand` tensor and produces a
 
 ```mlir
 // %operand: [[0.0, 1.0], [2.0, 3.0]]
-%result = "stablehlo.exponential"(%operand) : (tensor<2x2xf32>) -> tensor<2x2xf32>
-// %result: [[1.0, 2.71828183], [7.38905610, 20.08553692]]
+%result = "stablehlo.exponential"(%operand) : (tensor<2x2xf64>) -> tensor<2x2xf64>
+// %result: [[1.0, 2.7182818284590451], [7.3890560989306504, 20.085536923187668]]
 
 // %operand: (1.0, 2.0)
-%result = "stablehlo.exponential"(%operand) : (tensor<complex<f32>>) -> tensor<complex<f32>>
-// %result: (-1.13120438, 2.47172667)
+%result = "stablehlo.exponential"(%operand) : (tensor<complex<f64>>) -> tensor<complex<f64>>
+// %result: (-1.1312043837568135, 2.4717266720048188)
```

Comment:
Drop the second example? The checklist asks to keep just one example in the spec.

Reply:
Sure, fixed.